nncf/openvino/engine.py

```diff
@@ -68,7 +70,7 @@ def infer(
         output_data = {}
         for tensor, value in model_outputs.items():
             for tensor_name in tensor.get_names():
-                output_data[tensor_name] = value
+                output_data[tensor_name] = OVNNCFTensor(value)
```

Comment:
This is one such point for OV. It is enough to wrap the raw tensor value with a backend-specific derived class version of `NNCFTensor` in order for the resulting value to be usable in all common code based on `NNCFTensor` interface.